docs/reference/commands/cli-jvm-options.asciidoc

```diff
@@ -2,9 +2,10 @@
 [float]
 ==== JVM options
 
-CLI tools run with 64MB of heap. For most tools, this value is fine. However, if needed
-this can be overriden by setting the CLI_JAVA_OPTS environment variable. For example,
-the following increases the heap size used by the `pass:a[{tool-name}]` tool to 1GB.
+CLI tools run with 64MB of heap. For most tools, this value is fine. However, if
+needed this can be overriden by setting the CLI_JAVA_OPTS environment variable.
+For example, the following increases the heap size used by the
+`pass:a[elasticsearch-{tool-name}]` tool to 1GB.
```

Comment:
Out of interest what does the `pass:a` bit do?

Reply:
It tells Asciidoctor to substitute the `{tool-name}` attribute (variable) even though it's inside literal backticks.

There are some related docs here: https://docs.asciidoctor.org/asciidoc/latest/pass/pass-macro/